features/shop/checkout_with_paypal_payment.feature

```diff
@@ -0,0 +1,28 @@
+@ui-checkout @insulated @mink:firefox
+Feature: Checkout with PayPal Express Checkout
+    In order to buy products
+    As a Customer
+    I want to be able to pay with PayPal Express Checkout
+
+    Background:
+        Given the store is operating on a single channel
+          And default currency is "EUR"
+          And there is user "john@example.com" identified by "password123"
+          And store has a product "PHP T-Shirt" priced at "$19.99"
+          And store ships to "France"
+          And store allows paying "PayPal Express Checkout"
+          And store ships everything for free
+          And I am logged in as "john@example.com"
+
+    Scenario: Being redirected to the PayPal Express Checkout page
+        Given I added product "PHP T-Shirt" to the cart
+         When I proceed selecting "PayPal Express Checkout" payment method
+          And I confirm my order
+         Then I should be redirected to PayPal Express Checkout page
+
+    Scenario: Successful payment
+        Given I added product "PHP T-Shirt" to the cart
```

Comment:
I know this is confusing and we are still figuring out how to do it properly, but this could be a Given:

```
Given I have product X in the cart
When ...
```

Action starts when you "proceed" selecting X payment method, etc.

WDYT?